example-repo/.dmno/dmno-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+projects:
```

Comment:
not sure about this name...

- "packages" feels a bit too javascript-y
- "services" is our current nomenclature, although technically here we are saying where folders are that _might_ be dmno services, they still need a `.dmno` folder  to be considered a dmno service
- "workspaces" (npm, bun) is just confusing
- "projects" (moonrepo) feels fairly reasonable in a polyglot world


We could replace the term "service" with "project" everywhere?